addons/account/models/account_move.py

```diff
@@ -3087,6 +3087,8 @@ def _autopost_draft_entries(self):
             ('date', '<=', fields.Date.context_today(self)),
             ('auto_post', '=', True),
         ])
+        draft_records = records.filtered(lambda r: r.move_type != 'entry' and not r.partner_id or not r.invoice_line_ids)
```

Comment:
these conditions should be moved in the search

Reply:
Hello @agr-odoo 

https://github.com/odoo/odoo/blob/985d820ed64f90b8b827c5e6d2238a6fb1547a84/addons/account/tests/test_account_move_entry.py#L66
https://github.com/odoo/odoo/blob/985d820ed64f90b8b827c5e6d2238a6fb1547a84/addons/account/tests/test_account_move_entry.py#L88

If we move these conditions in the search, these two test cases will fail.
So should we change the search or test cases in 16.0?

Reply:
Hello @agr-odoo,
Changes have been done.
Thank you!